games-strategy/xbattleai/xbattleai-1.2.2-r3.ebuild

```
@@ -0,0 +1,52 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs autotools
+
+DESCRIPTION="A multi-player game of strategy and coordination"
+HOMEPAGE="https://inf.ug.edu.pl/~piotao/xbattle/mirror/www.lysator.liu.se/XBattleAI/"
+SRC_URI="https://inf.ug.edu.pl/~piotao/xbattle/mirror/www.lysator.liu.se/XBattleAI/${P}.tgz"
```

```suggestion
HOMEPAGE="https://inf.ug.edu.pl/~piotao/zasoby/xbattle/mirror/www.lysator.liu.se/XBattleAI/"
SRC_URI="https://inf.ug.edu.pl/~piotao/zasoby/xbattle/mirror/www.lysator.liu.se/XBattleAI/${P}.tgz"
```
Seems like the `HOMEPAGE` changed sometime ago, but you can still find it - download should work too.

---

Oh! That's very nice.